src/render-summary.js

```diff
@@ -27,7 +27,7 @@ export default class Summary {
 
         updateUi.call(this);
 
-        window.addEventListener('update', updateUi);
+        window.addEventListener('update', () => updateUi.call(this));
```

Comment:
Yup, this fix. appears in the pjo branch too (sorry for creating the bug).

Reply:
np)